oemer/bbox.py

```diff
@@ -28,7 +33,7 @@ def get_edge(data):
     return data
 
 
-def merge_nearby_bbox(bboxes, distance, x_factor=1, y_factor=1):
+def merge_nearby_bbox(bboxes: List[Tuple[int, int, int, int]], distance: float64, x_factor: int = 1, y_factor: int = 1) -> List[Tuple[int, int, int, int]]:
```

Comment:
Actually you could just annotate `distance: float` since the default python float is already float64.

Reply:
Or I guess there is some issue with mypy check to do in this way?